packages/core-components/src/lib/unsorted/ui/TelemetryOptOut/TelemetrySettingsPanel.svelte

```diff
@@ -31,7 +31,9 @@
 					<p>Each time you run a query, we get the following pieces of information:</p>
 					<ol class="list-decimal px-8">
 						<li>
-							A random identifier that is stored in <code>.evidence/template/.profile.json</code>
+							A random identifier that is stored in <code
+								>.evidence/customization/.profile.json</code
```

Comment:
This is probably an unintentional change from the formatter

Maybe won't make a difference